Xamarin.Essentials/Connectivity/Connectivity.shared.cs

```diff
@@ -9,12 +9,14 @@ public static partial class Connectivity
         static event EventHandler<ConnectivityChangedEventArgs> ConnectivityChangedInternal;
 
         // a cache so that events aren't fired unnecessarily
-        // this is mainly an issue on Android, but we can stiil do this everywhere
+        // this is mainly an issue on Android, but we can still do this everywhere
         static NetworkAccess currentAccess;
         static List<ConnectionProfile> currentProfiles;
 
         public static NetworkAccess NetworkAccess => PlatformNetworkAccess;
 
+        public static SignalStrength WiFiSignalStrength => PlatformSignalStrength();
```

Comment:
Should we be caching this as well to avoid unnecessary events?

Reply:
Which events could be triggered by this? I am mostly worried about iOS because I allocate a new NEHotspotNetwork on every call.